src/intro.md

```diff
@@ -237,6 +238,7 @@ Keep lines sorted.
 [ex-std-read-lines]: basics.html#ex-std-read-lines
 [ex-tar-compress]: app.html#ex-tar-compress
 [ex-tar-decompress]: app.html#ex-tar-decompress
+[ex-tar-iterate]: app.html#ex-tar-iterate
```

Comment:
please change the tag to reflect the example that we will have after its finished